com.chartboost.mediation/Documentation/integration/initialization.md

```diff
@@ -2,93 +2,104 @@
 
 ## Adding the import header
 
-Add the following import header to the top of any class file that will be using a Helium class.
+Add the following import header to the top of any class file that will be using a Chartboost Mediation class.
 
 ```c#
-using Helium;
+using Chartboost;
 ```
 
-## Initializing Helium SDK
+## Initializing Chartboost Mediation Unity SDK
 
-In order to initialize the Helium SDK, you will need your Helium App ID & App Signature. This can be obtained in your [Helium Dashboard](https://helium.chartboost.com).
-There are 2 ways you can go about providing your App IDs to the Helium SDK.
+In order to initialize the Chartboost Mediation UNity SDK, you will need your Chartboost Mediation App ID & App Signature. This can be obtained in your [Charboost Mediation Dashboard](https://helium.chartboost.com).
```

Comment:
```suggestion
In order to initialize the Chartboost Mediation Unity SDK, you will need your Chartboost Mediation App ID & App Signature. This can be obtained in your [Charboost Mediation Dashboard](https://helium.chartboost.com).
```